src/erlcloud_ddb2.erl

```diff
@@ -3140,6 +3149,238 @@ tag_resource(ResourceArn, Tags, Config) ->
       [{<<"ResourceArn">>, ResourceArn},
        {<<"Tags">>, dynamize_tags(Tags)}]).
 
+%%%------------------------------------------------------------------------------
+%%% TransactGetItem
```

Comment:
missing s here and G2G

Reply:
i'll fix forward this. we should not hold this just for the comment typo